frame/lottery/src/weights.rs

```diff
@@ -56,33 +56,33 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	fn buy_ticket() -> Weight {
-		(71_604_000 as Weight)
+		(72_703_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(6 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	fn set_calls(n: u32, ) -> Weight {
-		(15_015_000 as Weight)
+		(15_536_000 as Weight)
 			// Standard Error: 5_000
-			.saturating_add((301_000 as Weight).saturating_mul(n as Weight))
+			.saturating_add((786_000 as Weight).saturating_mul(n as Weight))
```

Comment:
This seems to be the difference with encoding? Double the weight per call

Reply:
we can't say for sure until we compare with the benchmark with the decoding but without the Box, but anyway this seems ok to me